{"name": "困人事件标记误报", "status": "passed", "attachments": [{"name": "log", "source": "ffbe1549-879c-44cd-8912-6e761df83dae-attachment.txt", "type": "text/plain"}], "start": 1692236262323, "stop": 1692236262403, "uuid": "e75836d5-eed7-45b5-964b-c0bc998c13c5", "historyId": "ab566412ce2c9ed8e210215e4ae8a5dd", "testCaseId": "16b1c3c86243e1b02c591065644e1507", "fullName": "opsPC.test_emergency#test_update_deal_status_false_alarm_001", "labels": [{"name": "feature", "value": "运维管理平台"}, {"name": "story", "value": "应急处置"}, {"name": "parentSuite", "value": "opsPC"}, {"name": "suite", "value": "test_emergency"}, {"name": "host", "value": "cale"}, {"name": "thread", "value": "13776-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "opsPC.test_emergency"}]}